pkgs/top-level/all-packages.nix

```diff
@@ -967,6 +967,8 @@ in
 
   aws-google-auth = python3Packages.callPackage ../tools/admin/aws-google-auth { };
 
+  aws-nuke= python3Packages.callPackage ../tools/admin/aws-nuke { };
```

Comment:
```suggestion
  aws-nuke = python3Packages.callPackage ../tools/admin/aws-nuke { };
```

Reply:
Why python3Packages when it's a go project?

Reply:
Fixed, thanks, good catch!